homeassistant/components/device_tracker/__init__.py

```diff
@@ -180,22 +175,6 @@ def async_setup_platform(p_type, p_config, disc_info=None):
 
     tracker.async_setup_group()
 
-    @callback
-    def async_device_tracker_discovered(service, info):
-        """Handle the discovery of device tracker platforms."""
-        hass.async_add_job(
-            async_setup_platform(DISCOVERY_PLATFORMS[service], {}, info))
-
-    discovery.async_listen(
-        hass, DISCOVERY_PLATFORMS.keys(), async_device_tracker_discovered)
-
-    @asyncio.coroutine
-    def async_platform_discovered(platform, info):
```

Comment:
This is still needed.

Reply:
@MartinHjelmare I think ```async_device_tracker_discovered()``` is not needed, because it was only used by `netgear` platform. What should be returned is `async_platform_discovered()`

Reply:
Yes, only add the code that starts on the lines where my comment starts, eg here that's 93. I haven't marked the import that's needed btw.